Upgrading libblkid1 from 2.30.2 to 2.31.1 causes cryptfs-tpm2 to error with "Too large count value passed to g_input_stream_read"

See https://github.com/jiazhang0/cryptfs-tpm2/issues/3 for details.  I've identified a specific commit in util-linux (https://github.com/karelzak/util-linux/commit/d2b0c6587d729a8e838821cb70d4de18c6969868) that appears to interfere with communication between cryptfs-tpm2 and tpm2-abrmd.  I'm unsure where this issue belongs.

With G_MESSAGES_DEBUG=all, on the cryptfs-tpm2 side I am getting:

```
Tue Aug 28 18:16:32 UTC 2018: [INFO] Use tabrmd as the default tcti interface
** (process:20547): DEBUG: initialized tabrmd TCTI context with id: 0x27698b7171430a6
** (process:20547): DEBUG: tss2_tcti_tabrmd_transmit
** (process:20547): DEBUG:     80 01 00 00 00 16 00 00 01 7a 00 00 00 05 00 00
** (process:20547): DEBUG:     00 00 00 00 00 01
** (process:20547): DEBUG: blocking write on iostream: 0x7f2028007ad0
** (process:20547): DEBUG: writing 22 bytes starting at 0x1372628 to socket 0x7f2033855ec0
** (process:20547): DEBUG: wrote 22 bytes to ostream 0x7f2028007ad0
** (process:20547): DEBUG: returning 22
** (process:20547): DEBUG: tss2_tcti_tabrmd_receive
** (process:20547): DEBUG: poll has 1 fds ready
** (process:20547): DEBUG:   POLLIN
** (process:20547): DEBUG: reading -20233408 bytes socket 0x7f2033855ec0, to 0x26be2e8

** (process:20547): WARNING **: read on istream 0x7f2028007b30 produced error: Too large count value passed to g_input_stream_read
** (process:20547): DEBUG: mapping errno 13 with message "Permission denied" to TSS2_TCTI_RC_GENERAL_FAILURE
Tue Aug 28 18:16:32 UTC 2018: [ERROR] Unable to get the TPM PCR banks (0xa0001)Tue Aug 28 18:16:32 UTC 2018: [ERROR] Unsupported PCR bank algorithm
** (process:20547): DEBUG: tss2_tcti_tabrmd_finalize
```

--- a/src/tcti-tabrmd.c
+++ b/src/tcti-tabrmd.c
@@ -390,6 +390,8 @@ tss2_tcti_tabrmd_set_locality (TSS2_TCTI_CONTEXT *context,
 void
 init_tcti_data (TSS2_TCTI_CONTEXT *context)
 {
+    memset (context, 0, sizeof (TSS2_TCTI_TABRMD_CONTEXT));
+
     TSS2_TCTI_MAGIC (context)            = TSS2_TCTI_TABRMD_MAGIC;
     TSS2_TCTI_VERSION (context)          = TSS2_TCTI_TABRMD_VERSION;
     TSS2_TCTI_TABRMD_STATE (context)     = TABRMD_STATE_TRANSMIT;

